Xamarin.Forms.Platform.iOS/Renderers/ActivityIndicatorRenderer.cs

```diff
@@ -4,6 +4,33 @@
 
 namespace Xamarin.Forms.Platform.iOS
 {
+	public sealed class UIActivityIndicatorViewDelegate : UIActivityIndicatorView
+    {
+        ActivityIndicator _element;
+        public UIActivityIndicatorViewDelegate(RectangleF point, ActivityIndicator element) : base(point)
+            => _element = element;
+
+        public override void Draw(CGRect rect)
+        {
+            base.Draw(rect);
+            if (_element?.IsRunning == true)
```

Comment:
This isn't called if it hasn't been triggered by LayoutSubviews() first. But we also need it here, because if you go away then come back again, it needs to call StartAnimating again to redraw it, because it stopped when you move away.